device/cluster.cpp

```diff
@@ -1751,11 +1753,11 @@ void* Cluster::host_dma_address(std::uint64_t offset, chip_id_t src_device_id, u
 }
 
 // Wrapper for throwing more helpful exception when not-enabled pci intf is accessed.
-inline PCIDevice* Cluster::get_pci_device(int device_id) const {
-    if (!m_pci_device_map.count(device_id)) {
+inline TTDevice* Cluster::get_tt_device(int device_id) const {
+    if (!m_tt_device_map.count(device_id)) {
         throw std::runtime_error(fmt::format("device_id: {} attempted to be accessed, but is not enabled.", device_id));
     }
-    return m_pci_device_map.at(device_id).get();
+    return m_tt_device_map.at(device_id).get();
```

Comment:
Since the `m_tt_device_map` is a `std::unordered_map<chip_id_t, std::unique_ptr<TTDevice>>`, does it make sense for this method to accept a `chip_id_t` instead of an `int`?